src/main/java/com/jfb/tickets/service/TicketService.java

```diff
@@ -19,33 +20,32 @@ public class TicketService {
     @Value("classpath:ticketData.txt")
     Resource resource;
```

Comment:
Didn’t notice last time, just to pay your attention: any variable on class level has to be with `public` or `private` modifier. When you choose `package private`  or  `package` modifier you need to have compelling reason for that.